<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VIAView</class>
 <widget class="QDialog" name="VIAView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>290</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>520</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">CS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BitsView" name="chipSelected" native="true">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string notr="true">PA</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="RegisterView" name="paView" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string notr="true">PB</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="RegisterView" name="pbView" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Counters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="timer1">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string notr="true">T1: 0000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timer1Latch">
            <property name="font">
             <font>
              <family>Monospace</family>
             </font>
            </property>
            <property name="text">
             <string notr="true">L: 0000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="timer2">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string notr="true">T2: 0000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timer2Latch">
            <property name="font">
             <font>
              <family>Monospace</family>
             </font>
            </property>
            <property name="text">
             <string notr="true">L:   00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>IRQ State</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="RegisterView" name="ierView" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="RegisterView" name="ifrView" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Auxiliary Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="RegisterView" name="acrView" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Peripheral Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="RegisterView" name="pcrView" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>98</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RegisterView</class>
   <extends>QWidget</extends>
   <header>views/RegisterView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BitsView</class>
   <extends>QWidget</extends>
   <header>views/BitsView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
